src/lib.rs

```diff
@@ -402,9 +402,9 @@ impl_slice_ranges! {&'a [u8]}
 #[macro_export]
 macro_rules! impl_find_token {
     ( $for_type:ty, $fragment_type:ty) => {
-        impl<'a, 'b> FindToken<LocatedSpan<$fragment_type>> for $for_type {
-            fn find_token(&self, input: LocatedSpan<$fragment_type>) -> bool {
-                self.find_token(input.fragment)
+        impl<'a, 'b> FindToken<$for_type> for LocatedSpan<$fragment_type> {
```

Comment:
Looks like it. I preferred swapping the macro's impl around rather than definitions here, but it might be more appropriate to restructure the macro here (`$fragment:ty, $token:ty`?). What do you think?